packages/builder-vite/README.md

```diff
@@ -104,6 +104,37 @@ module.exports = {
 };
 ```
 
+## TypeScript
+
+- Configure your `.storybook/main.ts` to use TypeScript
+
+```typescript
+import { StorybookViteConfig } from "@storybook/builder-vite"
+
+const config: StorybookViteConfig = {
+    ...,
+    viteFinal(config, options) {
```

Comment:
Let's make this async, to match with our other examples.

Reply:
Like `async viteFinal(...` ？

Reply:
Yep, exactly.